src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Http;
+using Raven.Client.Documents.Commands;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Server.Documents.Handlers.Processors.Replication;
+using Raven.Server.Documents.Operations;
+using Raven.Server.Documents.Sharding.Commands;
+using Raven.Server.Documents.Sharding.Operations;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.Documents.Sharding.Handlers.Processors.Replication
+{
+    internal class ShardedReplicationHandlerProcessorForGetConflicts : AbstractReplicationHandlerProcessorForGetConflicts<ShardedDatabaseRequestHandler, TransactionOperationContext>
+    {
+        public ShardedReplicationHandlerProcessorForGetConflicts([NotNull] ShardedDatabaseRequestHandler requestHandler)
+            : base(requestHandler, requestHandler.ContextPool)
+        {
+        }
+
+        protected override async Task GetConflictsByEtagAsync(TransactionOperationContext context, long etag)
+        {
+            var op = new ShardedGetReplicationConflictsOperation(RequestHandler.HttpContext, context, etag);
+            var conflicts = await RequestHandler.ShardExecutor.ExecuteParallelForAllAsync(op);
+
+            await WriteConflictsByEtag(context, conflicts, conflicts.Length);
+        }
+
+        protected override async Task GetConflictsForDocumentAsync(TransactionOperationContext context, string documentId)
+        {
+            var cmd = new GetDocumentConflictsCommand(RequestHandler, documentId);
+
+            var shardNumber = RequestHandler.DatabaseContext.GetShardNumber(context, documentId);
+            await RequestHandler.ShardExecutor.ExecuteSingleShardAsync(cmd, shardNumber);
+        }
+
+        private async Task WriteConflictsByEtag(JsonOperationContext context, BlittableJsonReaderObject[] conflicts, long totalResults)
+        {
+            await using (var writer = new AsyncBlittableJsonTextWriter(context, RequestHandler.ResponseBodyStream()))
+            {
+                context.Write(writer, new DynamicJsonValue
+                {
+                    ["TotalResults"] = totalResults,
+                    [nameof(GetConflictsResult.Results)] = conflicts
+                });
+            }
+        }
+
+        internal readonly struct ShardedGetReplicationConflictsOperation : IShardedOperation<BlittableArrayResult, BlittableJsonReaderObject[]>
+        {
+            private readonly HttpContext _httpContext;
+            private readonly TransactionOperationContext _context;
+            private readonly long _etag;
+
+            public ShardedGetReplicationConflictsOperation(HttpContext httpContext, TransactionOperationContext context, long etag)
+            {
+                _httpContext = httpContext;
+                _context = context;
+                _etag = etag;
```

Comment:
You don't use here pageSize/start + etag is local per database.
You will probably need to use the `ShardedPagingContinuation` here.
ping me about that.

Reply:
Let's block passing etag for the sharded EP entirely and use only the `ShardedPagingContinuation`.
note that we need allow paging here so don't throw on this function entirely.

Reply:
done